Remove extra "preview" word (#19708)
dotnet/docs: docs/core/porting/winforms.md

## Prerequisites

> - [Visual Studio 2019 version 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).

Author: ```suggestion
- [Visual Studio 2019 version 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&rel=16) for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).
```

> - [Visual Studio 2019 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).

Reviewer: ```suggestion
- [Visual Studio 2019 version 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).
```

- A working Windows Forms project in a solution that builds and runs without issue.
- A project coded in C#.

> > [!NOTE] <!-- TODO: (Need guidance on how this note should be changed) -->

Author: @OliaG, Any recommendation is this note?
It was added from a blog of you when version 16.5 was still in preview, that's outdated now.
Should the note be completely removed? or you do have another recommendation?

Reviewer 2: No I believe this remains, I checked the latest preview and the option is still listed under preview features, so I think it's good to stay for now. However, we can update references from core 3.0 to core 3.1

- A project coded in C#.

> [!NOTE]

> > .NET Core 3.0 projects are only supported in **Visual Studio 2019** or a later version. Starting with **Visual Studio 2019 version 16.5**, the .NET Core Windows Forms designer is also supported.

Reviewer: ```suggestion
> .NET Core Windows Forms projects are supported in Visual Studio 2019 and later versions. The .NET Core Windows Forms designer is supported starting in Visual Studio 2019 version 16.5.
```

I changed the wording to remove 3.0 completely. @adegeo Is this okay?
